test/public/envs/overview.test.js

```diff
@@ -119,6 +120,16 @@ module.exports = () => {
         await checkEnvironmentStatusColor(page, 3, 4);
     });
 
+    it('Should have correct row color in the overview page', async () => {
```

Comment:
I have not spotted that when we rebased it, but I think this test should be merged with the previous one (simply add the last line and delete completely the function `checkEnvironmentRowColor`)